CONTRIBUTING.MD

```diff
@@ -25,17 +30,27 @@ You can contribute in many ways: openning or participating of a issue, reporting
 
     Note, using a `git add .` will automatically add all files. You can do a `git status` to see your changes, but do it before `git add`.
 
-6.  Commit your changes using a descriptive commit message.
-
+6.  Commit your changes using [commitizen](http://commitizen.github.io/cz-cli/).
     ```sh
-    git commit -m "Brief Description of Commit"
+    yarn commitizen
```

Comment:
"yarn commitizen" (remember, before of to use commitizen, the packages should be installed previously)

WDYT?